src/main/java/com/unbappd/enterprise/dto/Review.java

```diff
@@ -0,0 +1,13 @@
+package com.unbappd.enterprise.dto;
+import lombok.Data;
+
+public @Data
+class Review {
+    private int reviewId;
+    private int placesId;
+    private String reviewText;
+    private int reviewScore;
+    private double longitude;
+    private double latitude;
+    private int reviewerID;
```

Comment:
Think about your structure you need to display nearby drinks to a user. Your google maps uses longitude and latitude to display locations, so adding it to Review may be helpful in the long run. You displayed reviewer names in your storyboard so I added reviewerID as some sort of connection to the user.